internal/hazelcast-client/backup_service.go

```
@@ -0,0 +1,46 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-platform-operator/internal/protocol/codec"
+	codecTypes "github.com/hazelcast/hazelcast-platform-operator/internal/protocol/types"
+)
+
+type BackupService struct {
+	Client ClientI
+}
+
+func NewBackupService(cl ClientI) *BackupService {
+	return &BackupService{
+		Client: cl,
+	}
+}
+
+func (bs *BackupService) ChangeClusterState(ctx context.Context, newState codecTypes.ClusterState) error {
+	req := codec.EncodeMCChangeClusterStateRequest(newState)
+	_, err := bs.Client.InvokeOnRandomTarget(ctx, req, nil)
+	if err != nil {
+		return fmt.Errorf("invoking: %w", err)
```

Do we add any information when we wrap error with `"invoking: %w"`? Can we just return `err`?

---

I agree we can just return the error